src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -132,6 +133,18 @@ export class QuickFixController extends Disposable implements IEditorContributio
 		this._ui.getValue().update(newState);
 	}
 
+	public hideCodeActionMenu() {
+		this._ui.getValue().hideCodeActionWidget();
```

Comment:
`getValue()` will cause the menu to be created if it does not already exist. Instead you should check `hasValue` first to avoid this